packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable camelcase */
```

Comment:
👀 
Why @ameerul-deriv? 🤔  
```suggestion
/* eslint-disable camelcase */
```

Reply:
it was because of all the snake_case values returned by the API but ill try and see if your suggestion below works